src/main/java/ch/ethz/idsc/retina/dev/rimo/RimoRateControllerDuo.java

```diff
@@ -6,14 +6,24 @@
 import ch.ethz.idsc.tensor.Scalar;
 import ch.ethz.idsc.tensor.Tensor;
 
+/** the "duo" controller was first used in the pure pursuit module
+ * for trajectory following. however, the analysis of the torque
+ * showed that the load was not balanced between the motors in a
+ * desirable way.
+ * 
+ * We learned that the torques assigned to both motors add up to
+ * an accumulated effect. Thus, a hidden imbalance can be created
```

Comment:
What do you mean exactly by "accumulated effect"? It seems from the plots actually that they add up to a constant amount always. This is shown in the first plot on page 4 in the document 20180128_rimo_status.pdf